scripts/new-dao.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const path = require('path');
+const libPath = path.join(__dirname, '..', 'lib');
+const addressesPath = path.join(libPath, 'addresses');
+
+const KreditsKit = artifacts.require('KreditsKit')
+
+module.exports = async function(callback) {
+  const networkId = parseInt(web3.version.network);
+
+  let kitAddresseFile = path.join(addressesPath, 'KreditsKit.json');
+  let kitAddresses = JSON.parse(fs.readFileSync(kitAddresseFile));
+  let kreditsKitAddress = process.env.KREDITS_KIT || kitAddresses[networkId]
+  if (!kreditsKitAddress) {
+    callback(new Error("KreditsKit address not found in environment variable KREDITS_KIT"))
+  }
+  console.log(`Using KreditsKit at: ${kreditsKitAddress}`);
+
+  let kreditsKit = KreditsKit.at(kreditsKitAddress)
+
+  kreditsKit.newInstance().then((ret) => {
+    console.log(ret.logs);
+    const installedEvents = ret.logs.filter(log => log.event === 'InstalledApp').map(log => log.args)
+    const deployEvents = ret.logs.filter(log => log.event === 'DeployInstance').map(log => log.args)
+
+    if (deployEvents.length > 1) {
+      callback(new Error("More than one DAO was deployed. Something is wrong"))
+    }
+    const daoAddress = deployEvents[0].dao;
+
+    let addresseFile = path.join(addressesPath, `dao.json`);
```

Comment:
Haha, make a function to update the `addresseFile`.

Reply:
yep, did that now finally. and added a `helpers' directory... 
do you have a good name for that file/function that injects that data based on the networkId?

Reply:
I was thinking about `updateAddresses` but as it also updates IDs I think `fileInject` works fine.